bb.edn

```diff
@@ -1,5 +1,5 @@
 {:tasks {dev (do (shell "yarn")
-               (shell "shadow-cljs -A:clerk-local server"))
```

Comment:
without it I get `Error building classpath. Local lib io.github.nextjournal/clerk.viewer not found: /home/mates/dev/clj/clerk/viewer`